gossip/src/crds_value.rs

```diff
@@ -204,6 +209,47 @@ impl SnapshotHash {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, AbiExample)]
+pub struct IncrementalSnapshotHash {
+    pub from: Pubkey,
+    pub base_slot: Slot,
```

Comment:
imo, `_slot` in `base_slot` is redundant. The type already says it is `: Slot,`.
same way that we do not say:
```rust
pub from_pubkey: Pubkey,
```

Reply:
Done!